assets/src/domain/eventEditor/services/apollo/mutations/datetimes/useOnDeleteDatetime.ts

```diff
@@ -31,7 +31,7 @@ const useOnDeleteDatetime = (): DatetimeMutationCallbackFn => {
 			});
 		}
 		// Update datetime cache after tickets cache is updated.
-		updateDatetimeCache({ proxy, datetimes, datetime, remove: true });
+		updateDatetimeCache({ proxy, datetimes, datetime, action: 'remove' });
```

Comment:
good one, `action: 'remove' is better than `remove: true`

Reply:
Yes, `action` is more clear